platform/util/pom.xml

```diff
@@ -26,5 +26,6 @@
     <modules>
         <module>platform-util</module>
         <module>platform-util-unavailableurls</module>
+        <module>uuidgenerator</module>
```

Comment:
nit: should this follow the `platform-util` naming that the other modules are using?